packages/open-flow/src/designer/browser/graph/ReactFlowContainer/ReactFlowContainer.module.scss

```
@@ -84,6 +84,10 @@
   }
 }
 
+.layoutPending :global(.react-flow__viewport) {
```

Stylelint (17.14.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)